src/Microsoft.Extensions.Logging/LogMessageFilter.cs

```diff
@@ -0,0 +1,4 @@
+namespace Microsoft.Extensions.Logging
+{
+    public delegate bool LogMessageFilter(string loggerType, string categoryName, LogLevel level);
```

Comment:
This seems more like a `LogCategoryFilter` or something like that - it doesn't touch the log message itself.